catalog/spatial/geocoding/spatial-geocoding-index/pom.xml

```diff
@@ -78,10 +92,17 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Export-Package>org.codice.ddf.spatial.geocoding.index</Export-Package>
+                        <Private-Package>
+                            org.codice.ddf.spatial.geocoding.query,
+                            org.apache.lucene.sandbox.queries
```

Comment:
If you're embedding lucene-queries, you don't also need to private package one of the packages from that jar. You already have the entire jar embedded.